src/main/net/polydawn/mdm/commands/MdmStatusCommand.java

```diff
@@ -52,6 +57,17 @@ public MdmExitMessage call() throws IOException {
 			assertInRepo();
 		} catch (MdmExitMessage e) { return e; }
 
+		if (depName != null) {
+			StoredConfig gitmodulesCfg = new FileBasedConfig(new File(repo.getWorkTree(), Constants.DOT_GIT_MODULES), repo.getFS());
+			try {
+				gitmodulesCfg.load();
+			} catch (ConfigInvalidException e) {
+				throw new MdmExitInvalidConfig(Constants.DOT_GIT_MODULES);
+			}
+			os.printf(MdmModuleDependency.load(repo, depName, gitmodulesCfg).getVersionActual());
```

Comment:
Review note: yes, `MdmModuleDependency.load` always returns non-nil.  Additionally, a thrown `MdmModuleTypeException` will be correctly caught at the top level of a CLI and a human-readable error message produced.  So this is all good.